config/locales/en/notifiers/course/assessment/answer/comment_notifier/replied/user_notifications/email.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  notifiers:
+    course:
+      assessment:
+        answer:
+          comment_notifier:
+            replied:
+              user_notifications:
+                email:
+                  subject: '%{course} New comment on %{topic}'
+                  message: >
+                    %{post_author} commented, %{post}
```

Comment:
`format_html` should not be used as that converts the HTML tags to stuff like `&gt;` for display. We do want this to be interpreted as HTML.